packages/solutions/app-tools/src/plugins/deploy/dependencies/index.ts

```diff
@@ -0,0 +1,262 @@
+import path, { isAbsolute } from 'node:path';
+import { fs as fse, pkgUp, semver } from '@modern-js/utils';
+import type { PackageJson } from 'pkg-types';
+import { readPackageJSON } from 'pkg-types';
+import { parseNodeModulePath } from 'mlly';
+import {
+  linkPackage,
+  writePackage,
+  isFile,
+  TracedPackage,
+  TracedFile,
+  findEntryFiles,
+  traceFiles,
+  findPackageParents,
+  resolveTracedPath,
+} from './utils';
+
+export const handleDependencies = async (
+  appDir: string,
+  serverRootDir: string,
+  include: string[],
+) => {
+  const base = '/';
+  const entryFiles = await findEntryFiles(serverRootDir);
```

Comment:
It seems static/ files are also in the directory, should it be 'entry' ?

Reply:
Yes, there's a problem here in the node.js scenario, let me fix it

Reply:
done~